docs/_newsfragments/1767.misc.rst

```diff
@@ -0,0 +1,4 @@
+Use ``Response`` media handlers when serializing to json ``HTTPError``
```

Comment:
Towncrier newsfragments are not subject to the docstring conventions; they should IMHO be worded as news items, telling what has happened.

Reply:
Capitalize JSON